### ISSUE TITLE
[Feature Request] Can we get access to text-multilingual-embedding models

### ISSUE THREAD

**Issue by Champkinz** [firebase/genkit#254]
**Is your feature request related to a problem? Please describe.**
I have a requirement to embed non English text, but at the moment i can only access textEmbeddingGecko through,
`import { textEmbeddingGecko, vertexAI } from '@genkit-ai/vertexai';`

**Describe the solution you'd like**
A way to use a multilingual embedding model through genkit

**Describe alternatives you've considered**
Using the [Vertex python SDK](https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-text-embeddings#generative-ai-get-text-embedding-python_vertex_ai_sdk)

I don't want mix too many SDKs, want to keep it concise so we can maintain our solution without a hassle.

**Comment by Champkinz** [comment]
So i did some digging and found this comment

```
/*
// @TODO(huangjeff): Fix multilingual text embedding gecko
// For some reason this model returns 404 but it exists in the reference docs:
// https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-text-embeddings

export const textEmbeddingGeckoMultilingual001 = embedderRef({
  name: 'vertexai/textembedding-gecko-multilingual@001',
  configSchema: TextEmbeddingGeckoConfigSchema,
  info: {
    dimensions: 768,
    label: 'Vertex AI - Multilingual Text Embedding Gecko',
    supports: {
      input: ['text'],
    },
  },
});
*/

export const textEmbeddingGecko = textEmbeddingGecko003;

export const SUPPORTED_EMBEDDER_MODELS: Record<string, EmbedderReference> = {
  'textembedding-gecko@003': textEmbeddingGecko003,
  'textembedding-gecko@002': textEmbeddingGecko002,
  'textembedding-gecko@001': textEmbeddingGecko001,
  //'textembeddding-gecko-multilingual@001': textEmbeddingGeckoMultilingual001,
};
```

I'll be working on fixing this or trying the defineEmbedder method

if successful, I can raise a PR for this.

**Comment by pavelgj** [comment]
These were added in #340 and released in 0.5.3.
